index.js

```diff
@@ -67,6 +103,27 @@ module.exports = function(app) {
     if (options.PSILCD1) {
       mapToNmea(PSILCD1)
     }
+        if (options.HDT) {
+      mapToNmea(HDT)
+    }
+    if (options.HDM) {
+      mapToNmea(HDM)
+    }
+    if (options.ROT) {
+      mapToNmea(ROT)
+    }
+    if (options.DBK) {
+      mapToNmea(DBK)
+    }
+    if (options.DBS) {
+      mapToNmea(DBS)
+    }
+    if (options.DBT) {
+      mapToNmea(DBT)
+    }
+    if (options.MTW) {
+      mapToNmea(MTW)
+    }
```

Comment:
This list is a bit stupid: we could iterate over `Object.keys(plugin.schema.properties)`.